scripts/sync-pull

```diff
@@ -11,7 +11,35 @@ sha1=${3}
 
 echo "sync for: $project $release"
 echo "********************************************"
-deb_cmd="ubuntu@chacra.ceph.com:/opt/repos/$project/$release/$sha1/debian/jessie/flavors/default/* /opt/repos/$project/$release/debian/jessie/"
+
+# This ugly loop check all possible DEB combinations to see which repo has the most packages since that's likely the repo you want to sync.
+current_highest_count=0
+for combo in debian/jessie debian/stretch ubuntu/trusty ubuntu/xenial ubuntu/bionic; do
+  combo_count=$(curl -s https://chacra.ceph.com/r/$project/$release/$sha1/${combo}/flavors/default/pool/main/c/ceph/ | wc -l)
+  if [ $combo_count -gt $current_highest_count ]; then
+    current_highest_count=$combo_count
+    highest_combo=$combo
+  fi
+done
+
+echo "Found the most packages ($current_highest_count) in $highest_combo."
+
+# Check the the DEB and RPM chacra endpoints to see if the repos are or need updating.
```

Comment:
this looks like a valid use case to add a number of packages in the repo. The repo API already knows this, it just doesn't expose it. It could appear in an endpoint like:

https://chacra.ceph.com/repos/ceph/mimic/1f43eda5fd672d639637e539f6a5015ee215c8d2/ubuntu/xenial/

This works for now though... good approach